lighthouse/classes/event_properties/definitions/barcode_no_plate_map_data.py

```diff
@@ -0,0 +1,22 @@
+from functools import cached_property
+from lighthouse.classes.event_properties.interfaces import EventPropertyAbstract
+from lighthouse.classes.event_properties.validations import SimpleEventPropertyMixin
+from lighthouse.constants.fields import FIELD_EVENT_BARCODE
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class BarcodeNoPlateMapData(EventPropertyAbstract, SimpleEventPropertyMixin):
+    def is_valid(self):
+        self.is_valid_param_not_missing(FIELD_EVENT_BARCODE)
+        return self._is_valid
+
+    @cached_property
```

Comment:
I think this is unnecessary because we're are getting a dictionary value which is super quick and cheap already:
> Useful for expensive computed properties of instances that are otherwise effectively immutable
https://docs.python.org/3/library/functools.html#functools.cached_property

Reply:
Yes, but the reason is this action has associated a retrieval of the value from a service most of the times, although not in this case in particular. This is to cache the response so we don't make several calls. These event properties objects are shared, so the answer from one could help to resolve the value on another.

Reply:
What I mean is, it needs to be cached, because otherwise we will send unneeded requests.